caravel/static/default.css

```diff
@@ -17,28 +17,14 @@ body {
 .branding {
     font: 400 30px Roboto, sans-serif;
     margin-top: 0.7em;
+    margin-right: 10px;
+    width: 170px;
+    display: inline-block;
 }
 
 .logo-image {
     height: 100px;
     display: inline;
-    margin-top: 10px;
-}
-
-.user-button-container {
-    padding-top: 0.5em;
-    padding-bottom: 0.5em;
-    text-align: right;
-}
-
-@media (min-width: 992px) {
-    .search-form {
```

Comment:
Well I need to remove user button container but since is cascading, i think it uses the .user-button-container from below.